### PR TITLE
Sample Basic Pipeline

### PR DESCRIPTION
**Pull request by ritesh97-rh**
Included jenkins project buildConfig, sample pipeline in Jenkinsfile, and README with detailed instructions on setup!

### CODE REVIEW

File: Pipeline/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+pipeline {
+    agent any
+    stages {
+       stage("Delete all test-app resources") {
+          steps {
+             echo "Hello! Pipeline is running! Resources are being deleted."
+             script {
+                openshift.withCluster() {
+                   openshift.withProject("test-app") { 
+		      openshift.selector("all", [ "app" : "test-app" ]).delete()
```
Comment (trevorbox) [inline Pipeline/Jenkinsfile]: Please replace "test-app" with a global variable in all locations.

File: Pipeline/jenkinsBuildConfig.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
```
Comment (trevorbox) [inline Pipeline/jenkinsBuildConfig.yaml:3]: only keep name in the metadata

File: Pipeline/jenkinsBuildConfig.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      uri: https://github.com/ritesh97-rh/OC-Test
+    type: Git
+  strategy:
+    jenkinsPipelineStrategy:
+      jenkinsfilePath: Pipeline/Jenkinsfile
+    type: JenkinsPipeline
+  successfulBuildsHistoryLimit: 5
+  triggers:
+  - type: ConfigChange
+status:
```
Comment (trevorbox) [inline Pipeline/jenkinsBuildConfig.yaml]: remove status block

File: README.md [hunk 4]
```diff
@@ -1,2 +1,45 @@
...
+##set up an app from jenkins-ephemeral template
+oc new-app jenkins-ephemeral 
+##to view available templates, run 'oc new-app --search jenkins'
+##apply the build config with name esri-example-pipeline to jenkins app (NOTE: replace {REPO_GITHUB_ADDR} with github repository)
+oc create -f Pipeline/jenkinsBuildConfig.yaml 
+##get route for Jenkins web console
+oc get route 
+##go to the route url to access Jenkins Web Console; on first log in, give Jenkins requested permissions
+
+2) Set up project to deploy C++ application using s2i
```
Comment (trevorbox) [inline README.md]: Step 2 should be done before Setting up Jenkins

File: README.md [hunk 5]
```diff
@@ -1,2 +1,44 @@
 # OC-Test
 OpenShift Testing
+
+Pipeline setup steps
+
+1) Set up Jenkins project  
+##create new project named test-app-pipeline
```
Comment (trevorbox) [inline README.md:17]: Looks like the markdown syntax isn't correct. Please use a markdown previewer.

Comment (ritesh97-rh) [reply README.md:17]: Fixed!

File: README.md (hunk 4 above)
Comment (ritesh97-rh) [reply README.md]: Not sure if it matters - I followed the steps in Chapter 8, section 2 of the do288 coursebook. Can change if it seems weird.

**Pull request closed without merging**